lib/Twig/Environment.php

```diff
@@ -758,7 +758,7 @@ public function compileSource($source, $name = null)
 
     public function setLoader(Twig_LoaderInterface $loader)
     {
-        if (!$loader instanceof Twig_SourceContextLoaderInterface && 0 !== strpos(get_class($loader), 'Mock_Twig_LoaderInterface')) {
+        if (!$loader instanceof Twig_SourceContextLoaderInterface && 0 !== strpos(get_class($loader), 'Mock_')) {
```

Comment:
Mock_Twig_LoaderInterface vs Mock_LoaderInterface when sing the namespaced variant
looking for mock_ is enough IMHO